pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientRecordEntity.java

```diff
@@ -32,9 +32,6 @@
 	@Column( name = "id", nullable = false )
 	private Long id;
 	
-	@Column(name = "organization_patient_record_id")
```

Comment:
we need this. Also getters and setters.
	@Column(name = "organization_patient_id")
	private String orgPatientId;